css/properties/font-size.json

```diff
@@ -0,0 +1,114 @@
+              "version_added": "1"
+            },
+            "chrome_android": {
+              "version_added": "1"
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "1"
+            },
+            "firefox_android": {
+              "version_added": "4"
+            },
+            "ie": {
+              "version_added": "5.5"
+            },
+            "ie_mobile": {
+              "version_added": "6"
+            },
+            "opera": {
+              "version_added": "7"
+            },
+            "opera_android": {
+              "version_added": "6"
+            },
+            "safari": {
+              "version_added": "1"
+            },
+            "safari_ios": {
+              "version_added": "1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "rem_values": {
+          "__compat": {
+            "description": "Rem values",
+            "support": {
+              "webview_android": {
+                "version_added": "4.1"
+              },
+              "chrome": {
+                "version_added": "31"
+              },
+              "chrome_android": {
+                "version_added": "42"
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": "12"
+              },
+              "firefox": {
+                "version_added": "31",
+                "notes": [
+                  "Before Firefox 57, animations using em units are not affected by changes to the <code>font-size</code> of the animated element's parent (<a href='https://bugzil.la/1254424'>bug 1254424</a>).",
+                  "Before Firefox 57, some language settings' inherited <code>font-size</code> is smaller than expected (<a href='https://bugzil.la/1391341'>bug 1391341</a>)"
+                ]
+              },
+              "firefox_android": {
+                "version_added": null
+              },
+              "ie": [
+                {
+                  "version_added": "11"
+                },
+                {
+                  "partial_implementation": true,
+                  "version_added": "9",
+                  "version_removed": "10"
+                }
+              ],
+              "ie_mobile": {
+                "version_added": "10"
+              },
+              "opera": {
+                "version_added": "28"
+              },
+              "opera_android": {
+                "version_added": "12"
+              },
+              "safari": {
+                "version_added": "7"
+              },
+              "safari_ios": {
+                "version_added": "4.1"
```

Comment:
I'm a little suspicious of this appearing on iOS so many versions before desktop, but then again I don't know how iOS and macOS version numbers line up, exactly.